plugins/task-plugin/src/task-plugin-backend.ts

```diff
@@ -0,0 +1,62 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+/**
+ * Generated using theia-plugin-generator
+ */
+
+import * as theia from '@theia/plugin';
+import * as che from '@eclipse-che/plugin';
+import 'reflect-metadata';
```

Comment:
I see that reflect-metadata is imported more than once in the plug-in while it shouldn't

Reply:
fixed, thank you!